MIGRATION.md

```diff
@@ -3,6 +3,10 @@
 API changes in MoveIt releases
 
 ## ROS Noetic
+- Added method `maximumRotation` to `RobotState`.
```

Comment:
please leave out that line. only things relevant during migration are added here.